docs/docs/05-reusable-components.md

```diff
@@ -232,3 +232,25 @@ Methods follow the same semantics as regular ES6 classes, meaning that they don'
 ### No Mixins
 
 Unfortunately ES6 launched without any mixin support. Therefore, there is no support for mixins when you use React with ES6 classes. Instead, we're working on making it easier to support such use cases without resorting to mixins.
+
+
+
+## Stateless Functions
+
+You may also define your React classes as a plain JavaScript function. For example using the stateless function syntax:
+
+```javascript
+function HelloMessage(props) {
```

Comment:
I agree at least showing ES5 example once is good. People often have all sorts of misconceptions about ES6. If the docs only show lambdas some people will assume they have to use lambdas.